### ISSUE TITLE
waveplot displays x-axis in time units but axis actually in downsampled sample index

### ISSUE THREAD

**Issue by justinsalamon** [librosa/librosa#470]
[waveplot](https://github.com/librosa/librosa/blob/c9c4254b540a8cf1091c6126c1be4890c86d298c/librosa/display.py#L254) produces a plot with time (seconds) as the unit for the x-axis, which makes it easy to think that overlaying another time series should align if it's x values are in time units too. However, the x-axis is actually in samples. Furthermore, by default the method downsamples the signal (for efficiency I imagine), which means that to align another time series one needs to compute its x positions in samples and then adjust these by the downsampling factor (by default 1000 / sr).

Basically right now plotting [requires a little bit of fidgeting](https://gist.github.com/justinsalamon/9ab86ef1c355ade1896bb91e69a2bca3) which can be confusing to less users who are less familiar with audio processing, and it might be easier to use this method if the x-axis was actually in time units?

I'm guessing using samples to index the x-axis was done for a reason, but I wonder what it is?

**Comment by bmcfee** [comment]
This issue was already resolved by #368.  All displays are now in natural coordinates.

Example notebook is [here](https://gist.github.com/a82ddecb576a039af85e303b2f2c51ec).
